functions/src/application/game/game.controller.ts

```diff
@@ -0,0 +1,50 @@
+import {Body, Controller, Inject, Param, Post} from '@nestjs/common';
```

Comment:
Sometime we have { dd } sometime {dd} we should use one way 🤔

Reply:
fixed